spec/index/index_spec.rb

```diff
@@ -280,4 +280,11 @@
       it { expect { idx.at 2, 3 }.to raise_error IndexError }
     end
   end
-end
\ No newline at end of file
+
+  context "Enumerable" do
```

Comment:
> The top level context is the method name in this case

I know this, but in this case I want to test `Daru::Index` to behave as like a Enumerable.
So I think we do not need to test all Enumerable method (`#map` and `#select` are sample methods to check `Daru::Index` is a Enumerable).

Do you think it is better to separate these test cases?